team-builder/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import './App.css';
+
+import teamData from "./teamData"
+import TeamHeader from "./components/TeamHeader";
+import TeamList from "./components/TeamList";
+import TeamForm from "./components/TeamForm";
+
+function App() {
+  const [teamList, setTeamList] = useState(teamData);
+  const [memberToEdit, setMemberToEdit] = useState(undefined);
```

Comment:
Curious why you instantiated your State to 'undefined', would null or just leaving it blank have sufficed?

Reply:
Yeah I think I could have left it out. It was pretty difficult implementing the edit functionality I was trying out different things to see if they would break things. In the end, memberToEdit is set to undefined after you're done editing a team card so I think I was testing if it would break things if memberToEdit was set to undefined from the beginning and it still worked.